src/Features/Core/Portable/Diagnostics/DiagnosticArguments.cs

```diff
@@ -71,6 +71,12 @@ internal class DiagnosticArguments
         [DataMember(Order = 8)]
         public IdeAnalyzerOptions IdeOptions;
 
+        /// <summary>
+        /// Indicates a high priority request, such as a user-invoked Ctrl + Dot operation to bring up the light bulb.
+        /// </summary>
+        [DataMember(Order = 9)]
+        public bool HighPriority;
```

Comment:
name seems good.  However, also consider using the name 'Explicit'.  We use that name in FindRefs to indicate that hte request came from an 'Explicit'ly invoked user request, as opposed to an ambiently invoked one (like taggers/pull-diags).  

This helps distinguish the cases, especially why implicit stuff should back off.

It also helps given that we already have the concept of diagnostic request priorities and analyzer priorities and code action priorities.  Calling this HighPriority is a bit confusing in the context of those other priority concept.s